MAUI/Rotator/Getting-started.md

```diff
@@ -0,0 +1,574 @@
+    using Microsoft.Maui.Hosting;
+    using Microsoft.Maui.Controls.Compatibility;
+    using Microsoft.Maui.Controls.Hosting;
+    using Microsoft.Maui.Controls.Xaml;
+    using Syncfusion.Maui.Core.Hosting;
+
+    namespace Rotator
+    {
+        public static class MauiProgram
+        {
+            public static MauiApp CreateMauiApp()
+            {
+                var builder = MauiApp.CreateBuilder();
+                builder
+                .UseMauiApp<App>()
+                .ConfigureSyncfusionCore()
+                .ConfigureFonts(fonts =>
+                {
+                    fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                });
+
+                return builder.Build();
+            }      
+        }
+    }   
+
+{% endhighlight %} 
+
+## Adding the .NET MAUI Rotator control
+
+Step 1: Add the NuGet to the project as discussed in the above reference section. 
+
+Step 2: Add the namespace as shown in the following code sample.
+
+{% tabs %}
+{% highlight xaml %}
+
+	xmlns:syncfusion="clr-namespace:Syncfusion.Maui.Rotator;assembly=Syncfusion.Maui.Rotator"
+
+{% endhighlight %}
+{% highlight c# %}
+
+	using Syncfusion.Maui.Rotator;
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Create a Simple SfRotator 
+
+The `SfRotator` control is configured entirely in C# code or by using XAML markup. The following steps explain on how to create a `SfRotator` and configure its elements,
+
+* Adding namespace for the added assemblies. 
+
+{% tabs %}
+
+{% highlight xaml %}
+
+	xmlns:syncfusion="clr-namespace:Syncfusion.Maui.Rotator;assembly=Syncfusion.Maui.Rotator"
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+	using Syncfusion.Maui.Rotator; 
+	
+{% endhighlight %}
+
+{% endtabs %}
+
+* Now add the `SfRotator` control with a required optimal name by using the included namespace.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <?xml version="1.0" encoding="utf-8" ?>
+    <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                xmlns:syncfusion="clr-namespace:Syncfusion.Maui.Rotator;assembly=Syncfusion.Maui.Rotator"
+                xmlns:local="clr-namespace:Rotator"
+                x:Class="GettingStarted.Rotator">
+    <ContentPage.Content>
+    <syncfusion:SfRotator x:Name="rotator" />	
+    </ContentPage.Content>
+    </ContentPage>
+	
+	
+{% endhighlight %}
+
+{% highlight C# %}		
+
+    using Syncfusion.Maui.Rotator;
+
+    namespace GettingStarted
+    {
+    public partial class RotatorControlPage : ContentPage
+        {
+            public RotatorControlPage()
+            {
+                InitializeComponent();
+                SfRotator rotator = new SfRotator();
+                this.Content = rotator;
+            }
+        }
+    }
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+
+## Add Rotator Items
+
+We can populate the rotator's items by using any one of the following ways,
```

Comment:
In getting started only use RotatorItem. In **populating items topic** covers the item template and item content topics